status.c

```diff
@@ -837,6 +846,9 @@ status_prompt_translate_key(struct client *c, key_code key, key_code *new_key)
 	}
 
 	switch (key) {
+	case KEYC_BSPACE:
+		*new_key = KEYC_LEFT;
+		return (1);
```

Comment:
Change `Backspace` in vi command mode to cursor left for consistency with vi and vi mode for popular shells like zsh(1) (bindkey -v) and bash(1) (set -o vi).